pandas/tests/groupby/aggregate/test_cython.py

```diff
@@ -343,9 +343,27 @@ def test_cython_agg_nullable_int(op_name):
         # the result is not yet consistently using Int64/Float64 dtype,
         # so for now just checking the values by casting to float
         result = result.astype("float64")
```

Comment:
Out of scope here, but I think this might pass now without the cast? Should probably be xfailed if not. I plan to look into this.